core/controllers/tasks_test.py

```diff
@@ -31,8 +36,8 @@
     [models.NAMES.job, models.NAMES.email])
 (feedback_models, suggestion_models) = models.Registry.import_models(
     [models.NAMES.feedback, models.NAMES.suggestion])
-taskqueue_services = models.Registry.import_taskqueue_services()
 transaction_services = models.Registry.import_transaction_services()
+platform_taskqueue_services = models.Registry.import_taskqueue_services()
```

Comment:
Why import this here if you don't use it?

Reply:
Removed.